unitex/build.xml

```diff
@@ -63,8 +63,9 @@
 
 	<target name="dist" depends="clean,compile" description="generate the distribution">
 		<!-- create Unitex.jar -->
-		<jar jarfile="${dist-dir}/Unitex.jar" manifest="classes/fr/umlv/unitex/Manifest.mf" basedir="classes" excludes="classes/fr/loria/**">
+		<jar jarfile="/Users/rojo/Unitex-GramLab-3.1/App/Unitex.jar" manifest="classes/fr/umlv/unitex/Manifest.mf" basedir="classes" excludes="classes/fr/loria/**">
```

Comment:
What is the motivation behind this change?

Reply:
I changed it because the application didn't compile in my computer if i use ${dist-dir}